Dockerfile

```diff
@@ -40,5 +40,5 @@ RUN make _test
 ##############
 # prod stage #
 ##############
-FROM base AS prod
+FROM registry.access.redhat.com/ubi9/python-312-minimal:9.6@sha256:706ed87c6b4b815d44a7cad8774b6c8403ce5a2e113934ceae30a8e807f568d4 AS prod
 COPY --from=builder /opt/app-root /opt/app-root
```

Comment:
won't we miss the license in this prod image?

Reply:
strange, nothing failed and konflux released image already, fix in https://github.com/app-sre/qontract-validator/pull/96